src/Playwright.NUnitTest/PlaywrightTest.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Concurrent;
+using System.Threading.Tasks;
+using System.Threading;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using Microsoft.Playwright;
+
+namespace Microsoft.Playwright.NUnitTest
+{
+    public class PlaywrightTest
+    {
+        private static Task<IPlaywright> playwrightTask_ = Microsoft.Playwright.Playwright.CreateAsync();
+
+        internal IPlaywright Playwright { get; private set; }
```

Comment:
```suggestion
        internal IPlaywright Playwright { get; }
```

Reply:
But I set it below?

Reply:
Sorry, I thought that was the constructor. All good then.